tests/suites/1_stateful/05_input_formats/05_0003_load_unload_all.result

```diff
@@ -0,0 +1,8 @@
+---CSV
+"a""b",1.0,"['a""b']","{""k"":""v""}","2044-05-06 10:25:02.868894"
```

Comment:
Why `\"` is shown as `""`. It looks a little strange.

Reply:
most CSV  use `""` instead of `\"` by default.